framework/generated/khronos_generators/khronos_base_generator.py

```diff
@@ -461,7 +461,38 @@ def beginFeature(self, interface, emit):
 
         # Some generation cases require that extra feature protection be suppressed
         if self.genOpts.protect_feature:
-            self.featureExtraProtect = self.get_feature_protect(interface)
+            self.featureExtraProtect = self.__get_feature_protect(interface)
```

Comment:
Where is this function defined?  It's not on dev, and it's not in this PR

Reply:
I inserted a raise Exception on the line above it, and it's never hit.  Are we sure we understand what is happening in this code, or are we missing a commit in the PR?

Reply:
The GitHub Ubuntu Release job runs the generators, which succeeded, and there was no difference with the output checked into the branch.  Is it possible this code isn't run any more?

Reply:
To add just a hair more context, `__get_feature_protect` wasn't executed as far back as ceecc6829, from Oct 2021.  So this is probably a function orphaned in some ancient PR that we should delete, but not as part of this PR.

Reply:
Got it. I've changed it back.